gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineClasspathConflictTest.groovy

```diff
@@ -0,0 +1,42 @@
+/*
+ * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline
+
+import com.palantir.baseline.plugins.BaselineClasspathConflict
+import org.gradle.api.Project
+import org.gradle.testfixtures.ProjectBuilder
+import spock.lang.Specification
+
+class BaselineClasspathConflictTest extends Specification {
```

Comment:
ideally the tests would show that this plugin finds a conflict when there is one and does not when there is not one

Reply:
I believe there are identically named classes in the following jars:

- https://mvnrepository.com/artifact/javax.el/javax.el-api/3.0.0
- https://mvnrepository.com/artifact/javax.servlet.jsp/jsp-api/2.1

e.g. `javax/el/ArrayELResolver.class`